### DISCUSSION TITLE
How to keep web application after not connect internet (offline) and refesh page?

### DISCUSSION THREAD

**Discussion by thanhpham288** [quasarframework/quasar#10095] [CLI - PWA mode]
I use PWA and IndexedDB but cannot keep web application after refesh page. Because I met Blank page when not connect internet (offline) and refesh page. Please guide to me to prevent blank page.

**Comment by ontwikkelfabriek** [comment]
@thanhpham288 

> I use PWA and IndexedDB but cannot keep web application after refesh page

I would run a Lighthouse report on your pwa site. (under chrome dev tools). It will report any pwa issues. 
Btw not all browsers support all pwa features ( like offline).

Btw this is one of my Quasar PWA app's that works offline. 
https://ipm-toolbox.nl

Does it run offline on your browser too? If not then there's something with your browser.

**Reply by thanhpham288** [reply]
I using Google Chrome, I think it support PWA

**Reply by ontwikkelfabriek** [reply]
@thanhpham288 

> I using Google Chrome, I think it support PWA

That's not what I asked...

**Reply by gonzalez-aldo** [reply]
What guide do you use to get your offline PWA?
